pkgs/modules/python/default.nix

```diff
@@ -18,10 +18,6 @@ let
       user = yes
       disable-pip-version-check = yes
       index-url = https://package-proxy.replit.com/pypi/simple/
```

Comment:
I am not ready to totally turn this off yet. Probably need to talk with the team. Can we disable for specific packages in https://github.com/replit/pip for now?

Reply:
i'd rather have python not be selectively broken for users. i think it's better to just broadly disable this and that way python Just Works on replit

Reply:
While we are at it, we can remove these lines:
```
      disable-pip-version-check = yes
      index-url = https://package-proxy.replit.com/pypi/simple/
```

Reply:
if we have an allowlist for the packages we know it works, it wouldn't be "selectively broken": things that we don't know whether it works for will succeed, and the things we know it succeeds for would also succeed!

this might be a good middle ground in the short / medium term.

Reply:
Yeah, I agree. Especially packages known to be large.